Library/Formula/osquery.rb

```diff
@@ -3,6 +3,7 @@
 class Osquery < Formula
   desc "SQL powered operating system instrumentation and analytics"
   homepage "https://osquery.io"
+  revision 1
```

Comment:
fixed in https://github.com/Homebrew/homebrew/commit/26d4f6e9253fd6c81c03ed40b6e30634331a49ac.